pkg/teeth/publish_logs_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package teeth
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestSourceQuery(t *testing.T) {
+	t.Parallel()
+	const (
+		testPullRequestEventsTable = "github_metrics_aggregator.pull_request_events"
+		testEventsTable            = "github_metrics_aggregator.events"
+		testLeechTable             = "github_metrics_aggregator.leech_status"
+		testInvocationCommentTable = "github_metrics_aggregator.invocation_comment_status"
+	)
+	want := `SELECT
```

Comment:
I would just use named parameters, really all this is testing is that the ordering of your %s replacements is correct, might be cleaner to avoid that failure scenario all together.